go/plek.go

```diff
@@ -44,8 +44,8 @@ var httpDomains = map[string]bool{
 // corresponding PLEK_SERVICE_FOO_URI environment variable.  For example, to
 // override the "foo-api" service url, set PLEK_SERVICE_FOO_API_URI to the base
```

Comment:
I'm not sure I follow.  This comment is describing how to override the base URL for a given service, using the "foo-api" service as an example.